src/plugins/blockresolver/README.md

```diff
@@ -32,43 +32,70 @@ A block consists of 2 parts:
 Currently the identifier must be unique.
  
 ## Example ##
-
-    % cat /tmp/test.block
-    bla
-    bla
-    bla
-    ### block config start
-    [section1]
-    key1 = val1
-    [section2]
-    key2 = val2
-    ### block config stop
-    asdf
-    asdf
-    asdf
-
-    % kdb mount -R blockresolver /tmp/test.block system/blocktest -c identifier="### block config"
-    % kdb export system/blocktest
-    [section1]
-    key1 = val1
-    [section2]
-    key2 = val2
-
-    % kdb set system/blocktest/section1/key12 val12
-    Create a new key system/blocktest/section1/key12 with string val12
-
-    % cat /tmp/test.block
-    bla
-    bla
-    bla
-    ### block config start
-    [section1]
-    key1 = val1
-    key12 = val12
-    [section2]
-    key2 = val2
-    ### block config stop
-    asdf
-    asdf
-    asdf
-
+```sh
+# Backup-and-Restore:system/examples/blockresolver
+sudo kdb mount -R blockresolver /tmp/test.block system/examples/blockresolver -c identifier=">>> block config" ini
+#
+# create testfile
+#
+$ echo "text" > /tmp/test.block
```

Comment:
i think keeping a way to write "real" comments would be important for longer examples and having both outputs and comments use the same syntax would lead to some parsing issues